src/renderer/viz/expressions/base.js

```diff
@@ -71,12 +69,25 @@ export default class Base {
         throw new CartRuntimeError('Unimplemented');
     }
 
+    /**
+     * Get the expression value
```

Comment:
We should document the potential case of failure (the expression is feature dependant)